src/status.ts

```diff
@@ -0,0 +1,281 @@
+import * as vscode from "vscode";
+
+import {
+  isGemMissing,
+  addGem,
+  bundleCheck,
+  bundleInstall,
+  isGemOutdated,
+  updateGem,
+} from "./bundler";
```

Comment:
I would find:
```suggestion
import Bundler from "./bundler";
```
and then referring to methods like `Bundler.addGem` to be much more readable then importing loose functions.